### PR TITLE
[5287] Bulk recommend - accordion page

### PR DESCRIPTION
**Pull request by jhenderson**
### Context
https://trello.com/c/QTZ0f0VF/5287-m-bulk-recommend-accordion-page

### Changes proposed in this pull request
- Accordian logic that breaks the number of rows into chucks of 50 if there's more that 50 rows to display
- New Javascript and CSS that allows the recommend button to remain visible at the bottom during scrolling

### Guidance to review
- Visit `/bulk-update/recommend/upload`
- Upload CSV
- Click button "Check who you’ll recommend"
- If number of trainees is less than 50, then no accordian breakdown is shown

### Screenshot
Screenshot 2023-03-14 at 17 37 44

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by jhenderson** [comment]
[[M] Bulk recommend - accordion page](https://trello.com/c/QTZ0f0VF/5287-m-bulk-recommend-accordion-page)

File: app/views/bulk_update/recommendations_checks/show.html.erb
Comment (jhenderson) [inline app/views/bulk_update/recommendations_checks/show.html.erb:15]: The number `50` is mentioned 4 times in this template. I have considered extracting - maybe settings or a constant. Holding off until I get a second opinion. There also might be a benefit of just keeping it as it is.

File: app/controllers/bulk_update/recommendations_checks_controller.rb
Comment (markhallen) [inline app/controllers/bulk_update/recommendations_checks_controller.rb:9]: Why do we need `count` here? `size` will fall back to `count` if the relation isn't already loaded.

File: app/views/bulk_update/recommendations_checks/show.html.erb
Comment (markhallen) [reply app/views/bulk_update/recommendations_checks/show.html.erb:15]: I'd lean towards extracting it, just because DRY 🤷‍♂️

**Review by markhallen — APPROVED** [review]
Looks great. Nice work

File: app/models/bulk_update/recommendations_upload_row.rb
Comment (d-a-v-e) [reply app/models/bulk_update/recommendations_upload_row.rb:30]: thanks @AbigailMcP was just puzzling over this in my PR 😆

**Pull request merged**
